README.md

```
@@ -50,19 +51,48 @@ The [Parse Server guide](https://docs.parseplatform.org/parse-server/guide/) is
+- Mount the cloud directory to access your Cloud Code files from the container
+
+You can customize the mounted volumes based on your needs, such as mounting config files or other directories that require persistence or runtime modifications.
+
 ## Helpful Scripts
 These scripts can help you to develop your app for Parse Server:
 
 * `npm run watch` will start your Parse Server and restart if you make any changes.
-* `npm run lint` will check the linting of your cloud code, tests and `index.js`, as defined in `.eslintrc.json`.
-* `npm run lint-fix` will attempt fix the linting of your cloud code, tests and `index.js`.
-* `npm run prettier` will help improve the formatting and layout of your cloud code, tests and `index.js`, as defined in `.prettierrc`.
+* `npm run lint` will check the linting of your cloud code, tests and `index.ts`, as defined in `.eslintrc.json`.
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: dash; Actual: asterisk

(MD004, ul-style)